gnmi_server/server.go

```diff
@@ -267,54 +247,129 @@ func (s *Server) checkEncodingAndModel(encoding gnmipb.Encoding, models []*gnmip
 	return nil
 }
 
+func ParseTarget(target string, paths []*gnmipb.Path) (string, error) {
```

Comment:
>ParseTarget [](http://example.com/codeflow?start=5&length=11)

Add unit tests for any parsing functions.

Reply:
updated